src/tests/util/github.rs

```diff
@@ -159,11 +167,25 @@ impl GitHubClient for MockGitHubClient {
             Err(not_found())
         }
     }
+
+    fn public_keys(&self, _username: &str, _password: &str) -> AppResult<Vec<GitHubPublicKey>> {
+        Ok(self
+            .data
+            .public_keys
+            .iter()
+            .map(|k| GitHubPublicKey {
+                key_identifier: k.key_identifier.to_string(),
+                key: k.key.to_string(),
+                is_current: *k.is_current,
+            })
```

Comment:
maybe implement `From< MockPublicKey>` for `GitHubPublicKey` below to make the conversion a bit easier :)